MyToDo/App/Category+CoreDataClass.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Category+CoreDataClass.swift
+//  MyToDo
+//
+//  Created by Ivan Ruiz Monjo on 20/2/21.
+//
+//
+
+import Foundation
+import CoreData
+
+
+public class Category: NSManagedObject {
```

Comment:
Why it's not bad to have a class file and a separate extension file, I personally like to merge them together. There's virtually no benefit from not doing that even though this is Xcode's default.